core/clingutils/CMakeLists.txt

```diff
@@ -119,7 +119,14 @@ foreach(file wchar.h bits/stat.h bits/time.h)
 endforeach()
 
 set(stamp_file ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/LLVMRES.stamp)
-add_custom_command(OUTPUT ${stamp_file}
+if(MSVC)
+  add_custom_command(OUTPUT ${stamp_file}
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_BINARY_DIR}/etc/cling/lib/clang/${LLVM_VERSION}/include
```

Comment:
Should we maybe move common lines out of the two blocks? I see `make_directory` is common; touching of the `stamp_file`; and the `DEPENDS`. I wouldn't even mind issuing the `COMMENT` for windows, even if it doesn't copy things (but creates a directory). If I'm not mistaken that leaves `copy_directory` protected by `MSVC` which is nice!

Reply:
You're right, well spotted! I'll fix it

Reply:
So for the record I tried, but CMake doesn't allow (doesn't understand) if()/endif() inside add_custom_command()...